samples/c2pa/C2paPlayer/C2paMenu/Providers.js

```diff
@@ -0,0 +1,42 @@
+// From the Verify codebase 2023
+//   ADOBE CONFIDENTIAL
+//   Copyright 2022 Adobe
+//   All Rights Reserved.
+//   NOTICE: All information contained herein is, and remains
+//   the property of Adobe and its suppliers, if any. The intellectual
+//   and technical concepts contained herein are proprietary to Adobe
+//   and its suppliers and are protected by all applicable intellectual
+//   property laws, including trade secret and copyright laws.
+//   Dissemination of this information or reproduction of this material
+//   is strictly forbidden unless prior written permission is obtained
+//   from Adobe.
+
+const matchers = [
```

Comment:
Are we just using this to get the application name?

Reply:
yep , is there a better way ?

Reply:
Oh this is for social, I guess not.